application/modules/quotes/models/Mdl_quotes.php

```diff
@@ -217,13 +217,8 @@ public function copy_quote($source_id, $target_id)
 
         // Copy the custom fields
         $this->load->model('custom_fields/mdl_quote_custom');
-        $db_array = $this->mdl_quote_custom->where('quote_id', $source_id)->get()->row_array();
-
-        if (count($db_array) > 2) {
-            unset($db_array['quote_custom_id']);
-            $db_array['quote_id'] = $target_id;
-            $this->mdl_quote_custom->save_custom($target_id, $db_array);
-        }
```

Comment:
@Verony-makesIT I think this part is also necessary and should be included. Am I missing something?

Reply:
@naui95 As I've tried to make clear before, I'm absolutely not a programmer and in this case I simply looked at the same function in **Mdl_Invoices.php**, then copied this code into **Mdl_Quotes.php** and tested it. 
Everything worked then without any problems and so I assumed that this is/was (?) the solution for this bug.
Sorry, but I have to admit that I don't have enough (developer) knowledge to answer this question. 
I could be completely wrong, but in my humble opinion this code is not necessary here because it also works in _**Mdl_Invoices.php**_, in the same way and without the "`if (count($db_array) > 2){ }`" code.